ts/private/ts_project.bzl

```diff
@@ -184,81 +184,110 @@ See https://github.com/aspect-build/rules_ts/issues/361 for more details.
     output_types = typings_outs + typing_maps_outs + typings_srcs
     use_tsc_noemit = ctx.attr.no_emit or (ctx.attr.transpile == 0 and not ctx.attr.declaration)
```

Comment:
this line can be removed since you're now setting this below

Reply:
Opps, that should have been dropped after my last refactor that was merged.

Reply:
Lets drop that before this PR: https://github.com/aspect-build/rules_ts/pull/701